### ISSUE TITLE
Some Issues 

### ISSUE THREAD

**Issue by JingyuLi-code** [layer6ai-labs/SGG-Seq2Seq#8]
Thank you for your code. when I run this repo, some issues confused me.

1. for code [https://github.com/layer6ai-labs/SGG-Seq2Seq/blob/main/transformer.py#L238](url) "relation_ids" contain relation label such as  tensor([[ 0, 48, 31, 48, 31, 31, 22, 31, 48, 48, 22], device='cuda:0'), "relation_ids" will embedded using nn.Embedding and fed into TripletEncoder. But, I think "relation_ids" are groundtruth, may be I'm wrong.

2.  I run the repo,  `python trainer.py --dataset vg --batch_size 512 --nhead 4 ` and get:
![image](https://user-images.githubusercontent.com/45377489/155981660-f551897d-783f-4491-abf6-420d2804c040.png)
    But, I find this repo doesn't use the visual feature of object regions? 

3. For union features in this repo called pairwise_features just contain box feature in [https://github.com/layer6ai-labs/SGG-Seq2Seq/blob/main/feature_utils.py#L65](url). Have you considered using visual union features like motif UnionBoxesAndFeats. Thanks.

**Comment by GalaxDust** [comment]
Hi ，  I can't find preprocess_evaluation.py and write_prediction.py, do u have the two files?
